spec/presenters/error_message_translator_spec.rb

```diff
@@ -67,171 +86,160 @@
     }
   end
 
-  let(:emt) { ErrorMessageTranslator.new(translations, key, error) }
+  it { is_expected.to respond_to(:long_message, :short_message, :api_message) }
```

Comment:
I don't think this is necessary as all these three methods are tested elsewhere.

Reply:
actually they are not explicitly tested anywhere so this is as much for documentary purposes IMO

Reply:
removed